package.json

```diff
@@ -7,32 +7,30 @@
     "type": "git",
     "url": "https://github.com/jbx-protocol/juice-contracts-v2"
   },
-  "version": "4.0.0",
+  "version": "5.0.0",
```

Comment:
I think a major version is the safest.

 `juice-interface` still compiles from this PR because we define all the necessary dependencies ourselves, but other consumers might have been relying on `ethers` from `@jbx-protocol/contracts-v2`; if they installed `@jbx-protocol/contracts-v2` again and didn't have their version locked, it would break their project.